### DISCUSSION TITLE
Agent does not do ReAct and not use tools

### DISCUSSION THREAD

**Discussion by ltm920716** [langgenius/dify#5147] [General]
hello，
  I try to build a Agent case，like below：
![image](https://github.com/langgenius/dify/assets/12007227/f29bc7dd-be8f-434b-a424-582da6484fc1)

but the ReAct is not working, so neither the log show
![image](https://github.com/langgenius/dify/assets/12007227/20ba427f-ff9e-46fa-ab60-4d8ede76e365)

![image](https://github.com/langgenius/dify/assets/12007227/c288c063-5ff9-4a6d-a36e-9a46682f98f4)

help pleaes, thanks!

**Comment by GlitCher50** [comment]
This is happening to me as well

![image](https://github.com/user-attachments/assets/02f0fa59-124c-4c79-a24e-4e0480efcd3b)


Also dying on me with no final answer 😅 

![image](https://github.com/user-attachments/assets/7d00b302-d269-4102-8ad6-a036390d8f4d)
Only tells it had used knowledge but no answer
